composer.json

```
@@ -16,6 +16,8 @@
         }
     },
     "require-dev": {
+        "roave/security-advisories": "dev-latest"
+        ,
```

🍺 🤓 Out of curiosity why we need this dependency?
```suggestion
        "roave/security-advisories": "dev-latest",
```

---

Its a safety check against dependancies with documented security issues, the PHP Security Advisories Database suggests it as best practice to be included. https://github.com/Roave/SecurityAdvisories https://phpqa.io/projects/roave-security-advisories.html#:~:text=Roave%20Security%20Advisories%20ensures%20that,the%20PHP%20Security%20Advisories%20Database.

---

Awesome, Thanks for explaining that 🙏